### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Joozty**

### CODE REVIEW

File: templates/data-providers/index.jsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import React, { useEffect, useRef, useLayoutEffect } from 'react'
 import { Button } from '@oacore/design'
+import countries from 'i18n-iso-countries'
+import englishLocale from 'i18n-iso-countries/langs/en.json'
```
Comment (viktor-yakubiv) [inline templates/data-providers/index.jsx:4]: I am afraid, we will hit the performance wall soon with this. Can you use the JSON only?

**Review by viktor-yakubiv — APPROVED** [review]
I don't like the fix and the reason it's caused but it's better than nothing. Thank you! 👍

File: templates/data-providers/index.jsx (hunk 1 above)
Comment (Joozty) [reply templates/data-providers/index.jsx:4]: Not really this should keep the bundle as small as possible.

Comment (viktor-yakubiv) [reply templates/data-providers/index.jsx:4]: How? Importing the library doesn't eliminate unused countries

Comment (Joozty) [reply templates/data-providers/index.jsx:4]: The bundle should contain only English names. And the rest of the library has like 300lines of code but I will change as we may spare those lines...

**Review by viktor-yakubiv — APPROVED** [review]
👍

**Pull request merged**